### DISCUSSION TITLE
UTM Windows 10 Arm Gets Stuck on setting location and Time

### DISCUSSION THREAD

**Discussion by kwaduo** [utmapp/UTM#3054] [UTM Help (macOS)]
I enter the time and location then the UTM gives me a 'Why did my PC Restart' error and continues in that loop forever. The error tells me that I should connect to wi-fi and to power and I have both of those things
Im using a MacBook Air M1
The VM is using the windows 10 install for ARM and is running QEMU 5.2

**Comment by conath** [comment]
I've moved this to Discussions since it's related to a VM guest instead of the UTM app itself.

Unfortunately I don't know how to fix your issue with Windows for ARM preview. Do note that, at this time, it is prerelease software and has more issues than the regular version of Windows 10. If you want to try that, I've made a [full tutorial](https://youtu.be/KsNoGPczavw).

If you want to keep trying with the ARM version, please provide more information so we can try to narrow down the issue. Is there an error code shown before the 'Why did my PC Restart' shows up? (a screen recording would also help)
Please also share the "QEMU command" so we can make sure you're using a recommended configuration.

<img width="794" alt="qemuarguments" src="https://user-images.githubusercontent.com/12073163/131038935-a6c7a19a-92c6-4164-ba7f-928a41222137.png">
